src/Networking/tests.cpp

```diff
@@ -125,7 +125,7 @@ TEST_CASE("Incremental PID", "[BaseStation]")
 
     REQUIRE(Globals::status_data["arm_base"]["operation_mode"] == "incremental PID speed");
     REQUIRE(Globals::status_data["arm_base"]["target_angle"] == start_angle);
-    int expected_increment = M_PI * 1000 / 8 / 10;
+    int expected_increment = 360 * 1000 * ((M_PI/8)/(2*M_PI)) / 10;
```

Comment:
Just curious, what does this expression mean in context? Isn't it equivalent to 1/16?

Reply:
Yeah. I give the incremental PID rate in terms of radians per second in `motor_interface.h` as `M_PI/8`.